profiles/coreos/arm64/package.accept_keywords

```diff
@@ -4,7 +4,7 @@
 =app-admin/sudo-1.8.20_p2 ~arm64
 =app-arch/bzip2-1.0.6-r8 ~arm64
 =app-arch/libarchive-3.3.1 ~arm64
-=app-crypt/gnupg-2.2.4 ~arm64
+=app-crypt/gnupg-2.2.4-r2 ~arm64
```

Comment:
I think this line can just be dropped.  That ebuild has both `amd64` and `arm64`.